docs/pages/dms/client.mdx

```diff
@@ -729,9 +797,31 @@ val client = Client().create(account = account, options = options)
 
 <div data-title="Swift">
 <table>
-<tr><th>Parameter</th><th>Default</th><th>Description</th><th></th></tr>
-<tr><td>appVersion</td><td>`undefined`</td><td>Add a client app version identifier that's included with API requests. **Production apps are strongly encouraged to set this value.**<br/>For example, you can use the following format: `appVersion: APP_NAME + '/' + APP_VERSION`.<br/>Setting this value provides telemetry that shows which apps are using the XMTP client SDK. This information can help XMTP developers provide app support, especially around communicating important SDK updates, including deprecations and required upgrades.</td><td></td></tr>
-<tr><td>env</td><td>`dev`</td><td>Connect to the specified XMTP network environment. Valid values include `.dev`, `.production`, or `.local`. For important details about working with these environments, see [XMTP `production` and `dev` network environments](#xmtp-network-environments).</td><td></td></tr>
+  <tr>
+    <th>Parameter</th>
+    <th>Default</th>
+    <th>Description</th>
+  </tr>
+  <tr>
+    <td>env</td>
+    <td>`dev`</td>
+    <td>Connect to the specified XMTP network environment. Valid values include `.dev`, `.production`, or `.local`. For important details about working with these environments, see [XMTP production and dev network environments](#xmtp-network-environments).</td>
+  </tr>
+  <tr>
+    <td>enableV3</td>
+    <td>`false`</td>
+    <td>To support group chat (a feature of XMTP v3), set to `true`. As a best practice, regardless of whether your app provides group chat, set `enableV3` to `true` to benefit from other aspects of the upgraded XMTP v3 protocol.</td>
+  </tr>
+    <tr>
+    <td>dbEncryptionKey</td>
+    <td>null</td>
+    <td>Optional. If not provided, the database remains unencrypted.</td>
```

Comment:
Required here as well. https://developer.apple.com/documentation/security/protecting-keys-with-the-secure-enclave